client/client_session.cpp

```diff
@@ -44,6 +44,16 @@ void nibaclient::client_session::handle_cmd(const std::string &input) {
         }
         std::vector<std::string> results;
         boost::split(results, input, boost::is_any_of("\t "));
+        if (results.empty())
```

Comment:
This makes sense. I'm curious why wasn't it here and what made you put it here now.